src/Plugin/StrawberryRunnersPostProcessor/OcrPostProcessor.php

```diff
@@ -474,6 +474,7 @@ protected function hOCRtoMiniOCR($output, $pageid) {
         $miniocr->startElement("b");
         $page->registerXPathNamespace('ns', 'http://www.w3.org/1999/xhtml');
         foreach ($page->xpath('.//ns:span[@class="ocr_line"]') as $line) {
+          $notFirstWord = 0;
```

Comment:
@giancarlobi simple solution thanks! @pcambra what do you think? maybe make $notFirstWord boolean directly? I have really no preference but may read "easier" if its FALSE, TRUE.

Reply:
@DiegoPino I fully agree, boolean seems really better choice

Reply:
Generally I like booleans better unless it's API/integration stuff